lnwire/peer_storage.go

```diff
@@ -0,0 +1,73 @@
+package lnwire
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+)
+
+// MaxPeerStorageBytes is maximum size in bytes of blob in peer storage message.
+const MaxPeerStorageBytes = 65531
+
+var ErrPeerStorageBytesExceeded = fmt.Errorf("peer storage bytes exceede" +
+	"d")
```

Comment:
use `errors.New` since there's nothing to format.

Reply:
May I know why you retracted it?